content/guides/basic-setup/index.md

```diff
@@ -13,48 +13,54 @@ If you require further help, please visit our forums or ask in our Discord's hel
 
 ## How to dump your PS2 BIOS
 
-Dumping your PS2 BIOS requires a mod to your PS2 to allow arbitrary code execution.
+Dumping your PS2 BIOS is conceptually a two-step process:
+1. Modify the operation of your PS2 so that it can run any program.
+2. Now you can run a "BIOS dumper" utility program on your PS2 that reads its BIOS and writes it to a USB drive.
 
-**Popular options:**
+There is a generally useful program, uLaunchELF, that lets you browse memory cards, DVDs, and USB drives connected to a PS and run programs from them. So in most of the approaches below, you "hack" your PS2 so that it runs uLaunchELF, and then use uLaunchELF to run the BIOS dumper utility from a USB drive.
```

Comment:
The second half of this is repetitive, you already said above that you have to modify your PS2 and run the bios dumper utility.

So maybe something like
> So for most of the approaches below, uLaunchELF is used as a means to run the BIOS dumper.

This also gets away from the "hack" mention that remains here.

Also change `PS` to `PS2`

Reply:
Done, I rephrased slightly.